assets/scss/app.scss

```diff
@@ -334,3 +334,12 @@ textarea {
     }
   }
 }
+
+//Cookie banner styling
+
+#cookie-banner {
+  #cookie-banner-actions {
+    font-size: 65% !important;
+    width: 66.667% !important;
+  }
+}
```

Comment:
can you explain why this is needed?

Reply:
The styling already existing in modern slavery impacts the cookie banner. Below is what it looks like without this new styling added

<img width="1151" alt="Screenshot 2021-02-26 at 11 16 28" src="https://user-images.githubusercontent.com/61828376/109293712-1451e800-7824-11eb-8160-c26177f9b39c.png">